scripts/testnet_checkpoint_bots.py

```diff
@@ -20,7 +20,6 @@
 from agent0.ethpy.base import smart_contract_transact
 from agent0.ethpy.hyperdrive import get_hyperdrive_pool_config
 from agent0.hyperfuzz.system_fuzz.run_fuzz_bots import _async_runner
```

Comment:
It seems like this `_async_runner` fn should not be private or owned by the fuzzing flow if we're using it for checkpoints on testnet.